### PR TITLE
HEP details for K8s

### PR DESCRIPTION
**Pull request by tmjd**
## Description
Addresses #873

This PR adds a new page to the "Kubernetes:Getting Started" outlining the unique policies that are needed when using Kubernetes and Host Endpoints

## Discussion points
- [ ] Is this the right place for the doc?
  - I could see it instead going under the Bare-metal section
- [ ] Is there anything that these policies open up that would not be wanted?
- [ ] What about things that not able to be blocked/restricted with this setup?

### CODE REVIEW

File: master/getting-started/kubernetes/host-endpoints.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
```
Comment (robbrockbank) [inline master/getting-started/kubernetes/host-endpoints.md]: This is a doc about host endpoints, but nowhere mentions any host endpoint config.  Would it be worth adding an example HEP yaml doc?

Also I assume this is for non-KDD backend, *and* the yaml docs below are calico yamls not k8s ones?

Comment (tmjd) [inline master/getting-started/kubernetes/host-endpoints.md]: Yep I was chatting with @fasaxc earlier and that was part of one of his suggestions to add a hep example.  I'll also add a note that this only applies to etcd (non-KDD) and must be configured with calicoctl.

**Comment by lxpollitt** [comment]
I think I should review this PR too before merging.

File: master/getting-started/kubernetes/host-endpoints.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+When [Using Calico to Secure Host Interfaces]({{site.baseurl}}/{{page.version}}/getting-started/bare-metal/bare-metal) 
+by setting up [Host Endpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/hostendpoint) 
+there are some additional considerations needed when using Kubernetes.
+
+### Allowing forwarded traffic in Host Endpoint Policy
+
+Since version v2.1.0 of Calico, Host Endpoint(HEP) policies apply to forwarded
+traffic.  Due to the forwarding that kubernetes sets up through kube-proxy,
+to handle its service IPs and NodePorts, it is necessary to add policy to allow
```
Comment (lxpollitt) [inline master/getting-started/kubernetes/host-endpoints.md]: I think the emphasis is wrong here.  It is not necessary to use host endpoints at all.  You can use host endpoints to restrict access to your cluster.  If you want to restrict access to only allow kubernetes node ports then you can use the following policy.  I would encourage the whole doc be oriented in this kind of +ve context.  Emphasis on what  the user is trying to achieve.

**Comment by tmjd** [comment]
With the pre-DNAT example now in Host Protection/Bare-metal docs, is this PR still needed? (Can the original issue be closed?)
Maybe just a link somewhere in the Kubernetes guides pointing to the pre-DNAT example might be enough.

**Comment by tmjd** [comment]
With Neil's worked example in the Host Endpoints docs a dedicated K8s page is not needed so I'm closing this PR.

**Pull request closed without merging**